bibtexparser/bparser.py

```diff
@@ -65,7 +65,8 @@ def __init__(self, data=None,
                  customization=None,
                  ignore_nonstandard_types=True,
                  homogenize_fields=False,
-                 common_strings=False):
+                 common_strings=False,
+                 add_raw_bibtex_entry=False):
```

Comment:
Where should I put the doc for this option? I tried to search for doc for common_strings or customization.
tutorial.rst with a custom section?